tt1-blocks/theme.json

```diff
@@ -11,6 +11,12 @@
 		}
 	],
 	"settings": {
+		"border": {
+			"customColor": true,
+			"customRadius": true,
+			"customStyle": true,
+			"customWidth": true
+		},
```

Comment:
Looks like border customizations are [already enabled](https://github.com/WordPress/theme-experiments/blob/203ee43174c61968b0d71111b38444aa199fbc7d/tt1-blocks/theme.json#L194) for the theme.

Reply:
😄 Thanks. I've removed that.